entrypoint.sh

```diff
@@ -2,13 +2,13 @@
 
 set -xe
 
-TFSEC_VERSION="latest"
+TFSEC_VERSION=""
 if [ "$INPUT_TFSEC_VERSION" != "latest" ]; then
-  TFSEC_VERSION="tags/${INPUT_TFSEC_VERSION}"
+  TFSEC_VERSION="/tags/${INPUT_TFSEC_VERSION}"
 fi
 
-wget -O - -q "$(wget -q https://api.github.com/repos/aquasecurity/tfsec/releases/${TFSEC_VERSION} -O - | grep -o -E "https://.+?tfsec-linux-amd64" | head -n1)" > tfsec-linux-amd64
-wget -O - -q "$(wget -q https://api.github.com/repos/aquasecurity/tfsec/releases/${TFSEC_VERSION} -O - | grep -o -E "https://.+?tfsec_checksums.txt" | head -n1)" > tfsec.checksums
+wget -O - -q "$(wget -q https://api.github.com/repos/aquasecurity/tfsec/releases -O - | grep -m 1 -o -E "https://.+?tfsec-linux-amd64" | head -n1)" > tfsec-linux-amd64
```

Comment:
I don't see how this will work when the user specifies a specific version of tfsec to use.

Reply:
Oh, yeah - I meant it just drop the `/`, not the whole thing.

Reply:
Fixed - sorry!